.eslintrc.cjs

```diff
@@ -32,5 +32,11 @@ module.exports = {
         'no-console': 'off',
       },
     },
+    {
+      files: ['packages/integrations/**/*.ts'],
+      rules: {
+        'no-console': 'off',
```

Comment:
Thoughts on making this something like

```js
'no-console': ['error', { allow: ['warn', 'error', 'info'] }],
```
That way we still catch random `console.log` or `console.table` while still allowing deliberate `warn`, `error` and `info`?

Reply:
That sounds good. Looks like we're using `console.log` and `console.debug` at:

https://github.com/withastro/astro/blob/5bcd76e3ab3dfaab1d84d0af46d7e5a55a2b6ce2/packages/integrations/image/src/utils/logger.ts#L71-L74

https://github.com/withastro/astro/blob/5bcd76e3ab3dfaab1d84d0af46d7e5a55a2b6ce2/packages/integrations/mdx/src/index.ts#L53-L61

I guess we can have `console.debug` allowed, and use line comments to suppress the second code as that's a temporary log only until mdx 1.0 (per the comment above the code)

Reply:
In my opinion, even considering that it's going away soon, those MDX `console.log` should probably all be changed to `console.info`

Reply:
I'll make it `console.info` then 👍 would be great to get rid of the eslint comments